src/czbenchmarks/tasks/single_cell/k562_perturbation.py

```diff
@@ -0,0 +1,288 @@
+from typing import List, Dict
+import pandas as pd
+import numpy as np
+import logging
+from ..task import Task, TaskInput, TaskOutput
+from ...tasks.types import CellRepresentation
+from ...metrics import metrics_registry
+from ...metrics.types import MetricResult, MetricType
+from ...constants import RANDOM_SEED
+
+from tqdm import tqdm
+
+logger = logging.getLogger(__name__)
+
+
+def binarize_values(y_true, y_pred):
+    ids = np.where(~np.isnan(y_true) & ~np.isinf(y_true))[0]
+    y_true = y_true[ids]
+    y_pred = y_pred[ids]
+    pred_binary = (y_pred > 0).astype(int)
+    true_binary = (y_true > 0).astype(int)
+    return true_binary, pred_binary
+
+
+class K562PerturbationTaskInput(TaskInput):
+    """Pydantic model for PerturbationTask inputs."""
+
+    de_res_wilcoxon_df: pd.DataFrame
+    pred_df: pd.DataFrame
```

Comment:
I think this should use the CellRepresentation type, but we may want to be more specific as I don't understand how the coordinates of the predicted values are being determined here. Let's discuss in standup today.